Makefile

```diff
@@ -358,7 +358,7 @@ jsonnet-vendor: $(JSONNET_BUNDLER) jsonnetfile.json jsonnetfile.lock.json
 JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
 
 .PHONY: jsonnet-format
-jsonnet-format:
+jsonnet-format: $(JSONNET)
```

Comment:
Does that target dependency actually install `jsonnetfmt`? It looks like it only installs `jsonnet`

Reply:
please wait ...

Reply:
@squat just find that jsonnetfmt seems cannot be installed by go install:
```
# go install 'github.com/google/go-jsonnet/cmd/jsonnetfmt'
go: finding github.com/google/go-jsonnet v0.14.0
go: downloading github.com/google/go-jsonnet v0.14.0
go: extracting github.com/google/go-jsonnet v0.14.0
can't load package: package github.com/google/go-jsonnet/cmd/jsonnetfmt: module github.com/google/go-jsonnet@latest found (v0.14.0), but does not contain package github.com/google/go-jsonnet/cmd/jsonnetfmt

```
Need more time to fix it.
How to mark this pr as draft?